OpenRA.Launcher/OpenRA.Launcher.csproj

```diff
@@ -65,4 +65,8 @@
       <Analyzer Remove="@(Analyzer)" />
     </ItemGroup>
   </Target>
+  <ItemGroup>
+    <TrimmerRootAssembly Include="netstandard" />
+    <TrimmerRootAssembly Include="System.IO.Pipes" />
```

Comment:
Do we need this line? Ie do we have anything dependent on pipes?

Reply:
Apparently so, since we crash without this... I don't know why the trimmer doesn't see this though, maybe referenced by one of our thirdparty deps loaded via a mod dll?

Reply:
What is the crash message? Also i cant find any dependency on pipes in our 3d party deps.

Reply:
```
Exception of type `System.Reflection.TargetInvocationException`: Exception has been thrown by the target of an invocation.
Inner
  Exception of type `System.IO.FileNotFoundException`: Could not load file or assembly 'System.IO.Pipes, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.

     at DiscordRPC.IO.ManagedNamedPipeClient..ctor()
   at DiscordRPC.DiscordRpcClient..ctor(String applicationID, Int32 pipe, ILogger logger, Boolean autoEvents, INamedPipeClient client)
   at OpenRA.Mods.Common.DiscordService..ctor(MiniYaml yaml) in /Users/paul/src/OpenRA/OpenRA.Mods.Common/DiscordService.cs:line 62
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at OpenRA.Manifest.LoadCustomData(ObjectCreator oc) in /Users/paul/src/OpenRA/OpenRA.Game/Manifest.cs:line 176
   at OpenRA.ModData..ctor(Manifest mod, InstalledMods mods, Boolean useLoadScreen) in /Users/paul/src/OpenRA/OpenRA.Game/ModData.cs:line 60
   at OpenRA.Game.InitializeMod(String mod, Arguments args) in /Users/paul/src/OpenRA/OpenRA.Game/Game.cs:line 446
   at OpenRA.Game.Initialize(Arguments args) in /Users/paul/src/OpenRA/OpenRA.Game/Game.cs:line 409
   at OpenRA.Game.InitializeAndRun(String[] args) in /Users/paul/src/OpenRA/OpenRA.Game/Game.cs:line 270
   at OpenRA.Launcher.Program.Main(String[] args) in /Users/paul/src/OpenRA/OpenRA.Launcher/Program.cs:line 30
```
Makes sense that this would be using a pipe.

Reply:
Then this line would only to be in the csproj that reference discordprc, right?

Reply:
Doesn't work. As far as I can tell `TrimmerRootAssembly` is ignored on anything except `<OutputType>Exe</OutputType>`